### PR TITLE
dvs_first_commit

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Biasing by greedy rewards is a game changer in many direct policy search applications.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Comment by teytaud** [comment]
It works, except with compiler-gym because impossible deep-copy.

File: nevergrad/benchmark/experiments.py [hunk 1]
```diff
@@ -1272,27 +1273,25 @@ def gym_problem(
...
-            else GymMulti(specific_problem, control=control, neural_factor=1 if control != "linear" else None, limited_compiler_gym=limited_compiler_gym)  # type: ignore
-            for control in ["neural", "linear"]
-        ]
+            else [
+                GymMulti(specific_problem, control=control, neural_factor=1 if control != "linear" else None, limited_compiler_gym=limited_compiler_gym, greedy_bias=greedy_bias)  # type: ignore
+                for control in ["neural", "linear"]
+            ]
+        )
     seedg = create_seed_generator(seed)
     optims = [
```
Comment (jrapin) [inline nevergrad/benchmark/experiments.py:1293]: only one left??

**Review by jrapin — COMMENTED** [review]
i don't get most of it, so go ahead if you are confident :s

File: nevergrad/benchmark/experiments.py (hunk 1 above)
Comment (teytaud) [reply nevergrad/benchmark/experiments.py:1293]: yes, I try to have everything reduced as far as possible...

**Comment by teytaud** [comment]
merged in newgym2

**Pull request closed without merging**